src/setupTests.ts

```diff
@@ -1,5 +0,0 @@
-// jest-dom adds custom jest matchers for asserting on DOM nodes.
-// allows you to do things like:
-// expect(element).toHaveTextContent(/react/i)
-// learn more: https://github.com/testing-library/jest-dom
-import '@testing-library/jest-dom/extend-expect';
```

Comment:
:/
Non so, temo che in futuro potrebbero tornarci utili, vabbe' comunque per il resto mi sembra tutto ok